BlueprintCore/BlueprintCore/Actions/Builder/ActionsBuilder.cs

```diff
@@ -151,7 +151,7 @@ public ActionsBuilder Add(GameAction action)
     /// <remarks>It is recommended to only call this when adding an action type not supported by the builder.</remarks>
     /// 
     /// <param name="init">Optional initialization <see cref="Action"/> run on the action.</param>
-    public ActionsBuilder Add<A>(Action<A>? init = null) where A : GameAction, new()
```

Comment:
Revert / sync to head where this is fixed. The generated code & template were out of sync. Same w/ ConditionsBuilder change.

Reply:
Fixed.